hazelcast-client/src/main/java/com/hazelcast/client/config/ClientSecurityConfig.java

```diff
@@ -16,6 +16,7 @@
 
 package com.hazelcast.client.config;
```

Comment:
this class has no API doc

Reply:
Let merge this right now if there is no extra problem and add the API doc later.

Reply:
Since we postponed the merge, I am adding the API doc on this pr.